source/AndroidXProject.cshtml

```diff
@@ -2,7 +2,7 @@
 @using System.Linq
 <Project Sdk="Xamarin.Legacy.Sdk">
   <PropertyGroup>
-    <TargetFrameworks>MonoAndroid9.0;net6.0-android30.0</TargetFrameworks>
+    <TargetFrameworks>MonoAndroid9.0;net6.0-android</TargetFrameworks>
```

Comment:
This change is missing in GPS-FB-MLKit PR

On purpose?

Reply:
https://github.com/xamarin/GooglePlayServicesComponents/pull/504#pullrequestreview-731591567